homeassistant/components/medcom_ble/manifest.json

```
@@ -0,0 +1,15 @@
+{
+  "domain": "medcom_ble",
```

```suggestion
  "domain": "medcom",
```

Unless there is plans on adding non-BLE devices we should use `medcom` since the namespace is available.

We only postfix when there is a namespace conflict

---

Medcom also makes non-BLE devices - USB or serial connected - which is why I used the suffix. So far it's better to use two different integrations for BLE and non-BLE devices even of the same brand, correct?

---

Should be fine to leave the `domain` as-is but I'd call the title `Medcom Bluetooth` since most users won't know what BLE is